lk2nd/device/dts/msm8953/apq8053-lenovo-cd-18781y.dts

```diff
@@ -0,0 +1,31 @@
+// SPDX-License-Identifier: GPL-2.0-only
+
+/dts-v1/;
+
+#include <skeleton64.dtsi>
+#include <lk2nd.dtsi>
+
+/ {
+	qcom,msm-id = <QCOM_ID_APQ8053 0>;
+	qcom,board-id= <0x1010520 0>;
+};
+
+&lk2nd {
+	model = "Lenovo ThinkSmart View";
+	compatible = "lenovo,cd-18781y";
+
+    lk2nd,dtb-files = "apq8053-lenovo-cd-18781y";
+
```

Comment:
> the only two buttons available on this device are Volume Up and Volume Down.

Then you may want to map vol+ to KEY_POWER keycode using gpio-keys, the menu loops around so it's fine but the message on the screen will be slightly incorrect. Better than having no menu though

Reply:
I've tried it and it doesn't actually work. Even with the `gpio-keys` configuration in there the Volume Up key still simply navigates through the menu.

Reply:
Ah, sorry, it's a subtle bug/quirk in the gpio-keys/menu implementation: it checks for the volume keys first and since there is no node for vol+, it uses the default key, which then suppresses the power keycode... 

As a hack, I think you can add a vol+ key with `gpios = <&pmic_pon 0 0>;` to suppress the keycode with an non-existing key, please add a HACK: comment near it if you do

Reply:
That worked, thanks!